.github/workflows/e2eForPR.yml

```diff
@@ -23,9 +23,44 @@ jobs:
             if [[ ${changed_file} =~ ^build\.ts$ ]]; then
               cantPassWithoutTests=true
             fi
+            if [[ ${changed_file} =~ src\/mockTests\.ts$ ]]; then
+                cantPassWithoutTests=true
+            fi
+            if [[ ${changed_file} =~ .*ci\.yml$ ]]; then
+                cantPassWithoutTests=true
+            fi
           done
           echo $cantPassWithoutTests
           echo CANT_PASS_WITHOUT_TESTS=${cantPassWithoutTests} >> $GITHUB_OUTPUT
+  build-and-test-e2e-mock:
+    runs-on: ubuntu-20.04
+    name: Test e2e for PR using mock app
```

Comment:
Just to confirm: the job with the id `build-and-test-e2e` and name `Test e2e for PR` is a required check, but the job with the id `build-and-test-e2e-mock` and name `Test e2e for PR using mock app` is _not_ a required check, correct?

Reply:
Yep, exactly!